chart/flux/CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 0.9.1 (2019-04-17)
+
+### Improvements
+
+ - Switch image registry from Quay to Docker Hub
```

Comment:
This is not accurate, I made a change to the CRD for the Helm operator refactor.